_includes/misc/beta-note.md

```diff
@@ -1 +1 @@
-> **Important**: {{beta_feature}} is currently in {{pre-release_type}}. To learn about our pre-release commitment, please read our [software licence agreement](https://about.tiny.cloud/legal/tiny-self-hosted-software-license-agreement-oem-saas/).
+> **Important**: Real Time Collaboration beta is invite only, [follow these steps]({{site.baseurl}}/rtc/beta-access/) to get started.
```

Comment:
This is a shard note, so we can't do this... (eg it's used on the exports page as well)

Reply:
reverted by https://github.com/tinymce/tinymce-docs/pull/1801/commits/b1c8bf02e467e8b197c433163c57f9ddc977262b

Reply:
That doesn't work, make a separate note the RTC beta then, and link back to the beta-access page. No need to have license link here, we have license checks in cloud signup as well as beta signup.